src/TemplateProcessor.ts

```diff
@@ -179,11 +181,11 @@ export default class TemplateProcessor {
         this.changeCallbacks = new Map();
     }
 
-    private setupContext(context: {}) {
+    private setupContext() {
```

Comment:
please add TS typings (parameters, return type)

Reply:
added return type (void). Should I somehow indicate that the function does not take any arguments?